gnd/src/main/java/com/google/android/gnd/model/form/Option.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.gnd.model.form;
+
+import com.google.auto.value.AutoValue;
+import javax.annotation.Nullable;
+
+/** Describes a single valid option to a multiple choice question. */
+@AutoValue
+public abstract class Option {
```

Comment:
You could also name this `Choice` if `Option` feels too close to `Optional`. I'm fine with it as is

Reply:
To me "Choice" is something that has been taken, whereas an "Option" is something available to the user. N'est-ce pas?

Reply:
That's a good distinction. In certain usages, the terms do blend, (for example, one says "what choice do I have", "multiple choice"), but you're right, in the object position, they designate different things "I disagree with his choice" is not equivalent to "I disagree with his option" even though "What choice do I have?" is nearly equivalent to "What are my options?"...language :dizzy_face: . We should stick to more rigorous usage you suggest. No other alternatives immediately come to mind. Even if it's not optimal, I think `Option` is good for now!

Reply:
Then again, option also typically implies a human agent behind it, whereas choice (when they serve a similar function) usually doesn't. 

Shades :sunglasses:  of meaning